spec/spec_helper.rb

```diff
@@ -31,6 +31,8 @@
 
   config.order = :random
   Kernel.srand config.seed
+
+  config.example_status_persistence_file_path = "spec/examples.txt"
```

Comment:
Good idea, though it could probably do with a more descriptive file name.

Reply:
pretty sure I got it from the RSpec docs.   https://relishapp.com/rspec/rspec-core/docs/command-line/only-failures

It was helpful when I was writing tests. That said, I'm indifferent to the naming.

Reply:
Sorry my bad, I didn't see that they failures are actually called 'examples' :)